src/js/components/MaskedInput/MaskedInput.js

```diff
@@ -185,6 +185,7 @@ const MaskedInput = forwardRef(
     const [value, setValue] = formContext.useFormInput({
       name,
       value: valueProp,
+      initialValue: '',
```

Comment:
Should we take `defaultValue` into account here at all?

Reply:
> Should we take `defaultValue` into account here at all?

What defaultValue? MaskedInput doesn't have a defaultValue prop. I think at best the caller would pass that in as a `value`

Reply:
TextInput supports a `defaultValue` -- I'm assuming the "rest" from MaskedInput ends up on the TextInput it's built with. https://github.com/grommet/grommet/blob/master/src/js/components/TextInput/TextInput.js#L477

In some cases like in CheckBoxGroup we do something like `initialValue: defaultValue || []` so wasn't sure if that was important to consider here or not. https://github.com/grommet/grommet/blob/master/src/js/components/CheckBoxGroup/CheckBoxGroup.js#L45

We don't have `defaultValue` documented on MaskedInput at the moment so I would be fine leaving it separate.

Reply:
> TextInput supports a `defaultValue` -- I'm assuming the "rest" from MaskedInput ends up on the TextInput it's built with. https://github.com/grommet/grommet/blob/master/src/js/components/TextInput/TextInput.js#L477
> 
Interesting. Note that it's not documented for TextInput either (https://v2.grommet.io/textinput) and it's not in TextInput's propTypes or typescript from what I can tell. I could add it to the formInput initialValue though

Reply:
> TextInput supports a `defaultValue` -- I'm assuming the "rest" from MaskedInput ends up on the TextInput it's built with. https://github.com/grommet/grommet/blob/master/src/js/components/TextInput/TextInput.js#L477

And actually it doesn't look like MaskedInput uses TextInput. It passes `rest` along to `StyledMaskedInput` which is just a `styled.input`

So I think we're good here (although poor TextInput probably should have defaultValue documented.)